### PR TITLE
Renderboard

### PR DESCRIPTION
**Pull request by samgaco**
Submitting for a code review.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+$color4: rgba(251, 139, 36, 1);
+$color5: rgba(227, 100, 20, 1); */
+
+/* .try{
+background: rgba(237, 62, 9, 1);
+background: rgba(129, 193, 75, 1);
+color: rgba(46, 147, 60, 1);
+color: rgba(41, 112, 69, 1);
+color: rgba(73, 79, 75, 1);
+} */
```
Comment (JamesOkunlade) [inline css/styles.css]: Don't leave unused code as a comment in your codebase. Remove them if they are not needed.

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+  
+}
+
+/* input[type='submit']{
+  width: 50vw;
+  margin-left: 15vw;
+  background-color: rgba(95, 15, 64, 1);
+  border-radius: 1em;
+  color: #fff;
+} */
```
Comment (JamesOkunlade) [inline css/styles.css]: Don't leave unused code as a comment in code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+  <div class="winmsg">
+    <p> <b></b> </p>
+  </div>
+
+  <footer> <i>Project by Simon Wathigo & Samuel Garcia. Microverse.inc @2019</i> </footer>
+  <script src="./js/script.js"></script>
+  <!-- <script src="./js/display_controller.js" type="module"></script>
+    <script src="./js/game_board.js" type="module"></script>
+    <script src="./js/player.js" type="module"></script> -->
```
Comment (JamesOkunlade) [inline index.html]: Remove this as well.

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+  const resetBoard = () => {
+    board = ['1', '4', '7', '2', '5', '8', '3', '6', '9'];
+    return board;
+  };
+  return {
+    board,
+    addPosition,
+    resetBoard,
+  };
+})();
```
Comment (JamesOkunlade) [inline js/script.js]: As I have summarized in my first point of the general review, this should be in a file of its own. You can then import it into any file you want to call it from.

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
# Changes required 😁 

**Good job so far. Here are some changes you'll need to implement to get a thumb up 🙂** 

- Separate your different modules into different files. `Gameboard`, `Player` and a `domManager`. You can then import different files in the different places you need them. Ensure to use `ES6 import` statement rather than `require`. 
- The essence of the `domManager` is to manipulate the dom. So you shouldn't mix your game logic with any dom-manipulation task. All dom-manipulation tasks should be done in the `domManager` only.
- I left some comments on your code as well, implement the changes I requested there too.

**Please implement those and request a re-review when you're done. 😄** 

**James Okunlade**
**Happy coding! 😎**

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
# Changes required 😁 

An error is preventing your game from starting. Check your `console` to see and fix the error.

Request for another review when you're done.

**James Okunlade**
**Happy coding!**

**Comment by samgaco** [comment]
Hi, Would it be possible to get some indications? We are not sure about how to solve this problem or what is causing the errors.

**Comment by JamesOkunlade** [comment]
image
There you go.

**Comment by samgaco** [comment]
We get the same errors and googling, those are gone by running a [server](https://jasonwatmore.com/post/2016/06/22/nodejs-setup-simple-http-server-local-web-server), once we run the server we get other errors that we are clueless on how to solve. Should the project be able to be launched with no local server?

**Review by JamesOkunlade — APPROVED** [review]
# Approved 👍 😄 

**Go ahead and merge. Good luck with your future projects. 😎** 

**James Okunlade**
**Happy coding! 😎**

**Comment by wathigo** [comment]
Thank you @JamesOkunlade

**Pull request merged**